### ISSUE TITLE
!mlflow ui is giving serving on http:/ DESKTOP-GI2PLS2:5000 in pycharm

### ISSUE THREAD

**Issue by ashishtele** [mlflow/mlflow#3912]
Configuration:

mlflow==1.13
Installed as a part of pycaret

Problem:
when I am running the command "!mlflow ui", it is giving the serving address as "http:/ DESKTOP-GI2PLS2:5000" instead of "http://localhost:5000/#/". It was working till yesterday (mlflow==1.12.1). How to resolve?

**Comment by harupy** [comment]
What happens if you downgrade mlflow?

pip install mlflow==1.12.1

**Comment by ashishtele** [comment]
@harupy 
Hi,
I tried both versions (1.12.1 and 1.13). The problem still persists.

**Comment by harupy** [comment]
Could you run this command `mlflow ui --host 0.0.0.0`?

**Comment by ashishtele** [comment]
Tried with it. Still the same result. Please find the attached screenshot.
![issue](https://user-images.githubusercontent.com/14126898/103185613-25fe2a00-488b-11eb-9501-e938219ea1b0.PNG)

**Comment by harupy** [comment]
What does `which mlflow` print out?

**Comment by ashishtele** [comment]
It is giving 'invalid syntax' error. 'which mlflow' is this correct?

**Comment by harupy** [comment]
It's a shell command. Could you try this `!which mlflow`?

**Comment by ashishtele** [comment]
I am running this code on Windows. Can I try with 'where mlflow'? referred it: https://www.shellhacks.com/windows-which-equivalent-cmd-powershell/

**Comment by harupy** [comment]
Sorry, just never mind my comment here: https://github.com/mlflow/mlflow/issues/3912#issuecomment-751553987.

**Comment by harupy** [comment]
Can you check the version of `waitress` (which should have been installed with `mlflow`)?

```sh
pip freeze
```

should print out a list of pacakges installed in your environment.

**Comment by ashishtele** [comment]
The installed version of waitress is 1.4.4. (Upgrade is available - 2.0.0b1)
![waitress](https://user-images.githubusercontent.com/14126898/103295876-e76b8b00-49c2-11eb-8166-a8dc779140e4.PNG)

**Comment by harupy** [comment]
@ashishtele 
Found a related issue in the waitress repo:

https://github.com/Pylons/waitress/issues/167

**Comment by harupy** [comment]
@ashishtele When you run `mlflow ui`, is the MLflow UI available on `localhost:5000`?

**Comment by ashishtele** [comment]
@harupy ..Not yet..I followed the Pylons/waitress#167 issue..but it does not have any relevant solution

**Comment by harupy** [comment]
@ashishtele Thanks for the reply. 

Sorry I should have been more clear. The comment below indicates that the UI is running on `localhost:5000` even if the console output says it's not. Could you open `localhost:5000` after running `mlflow ui` and check the UI is available there?

> WORKSTATION is my PC name. Entering the above url in my browser results in the This site can’t be reached error. Manually typing localhost:6543 in my browser works fine.

https://github.com/Pylons/waitress/issues/167#issue-237542958

**Comment by ashishtele** [comment]
@harupy ..UI is available at localhost:5000 but it does not show any active model run there. Previous runs were giving proper model records at localhost:5000.

**Comment by harupy** [comment]
Does an `mlruns` directory exist in your current working directory?

**Comment by ashishtele** [comment]
@harupy ..Yes, mlruns directory exist in current working directory. I think I figured it out. Though it is showing local_machine:5000, localhost:5000 shows the MLflow setup. When I expand the left hand side pan, I can see the experiemt_name I provided in the PyCharm setup. The default MLFlow view does not show the experiment I was running. It seems to be some web page alignment/ resolution related. 

I think we can close this thread. Thank you for your support.
